functions.php

```diff
@@ -123,6 +123,28 @@ function get_parallax_page_header($page_id) {
 	return ob_get_clean();
 }
 
+function get_page_video_markup( $page_id ) {
+	$video = null;
+
+	$video_id = get_post_meta( $page_id, 'page_video', TRUE );
+
+	$video = wp_get_attachment_url( $video_id ) ?: null;
+
+	ob_start();
+	if ( $video ) :
+?>
+	<video class="page-header-video" muted loop autoplay>
```

Comment:
The `<video>` elem is going to need the `data-object-fit="cover"` attribute for the object-fit polyfill to take effect.